cfgov/v1/util/migrations.py

```diff
@@ -0,0 +1,59 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+from treebeard.mp_tree import MP_Node
+
+
+def get_page(apps, slug):
+    base_page_cls = apps.get_model('wagtailcore', 'Page')
+    return base_page_cls.objects.get(slug=slug)
+
+
+def get_free_path(apps, parent_page):
```

Comment:
This terribleness is mainly due to the fact that you [can't really](http://stackoverflow.com/a/33807804) just do `parent_page.add_child` as we might like. So instead of doing that, you have to create the model instance in the DB with the proper fields, one of which is this `path` value. Pages in Wagtail have paths like `00010003000L` (note use of non-hexadecimal character) so this function just chooses a valid path for each new page being added.